flask_rebar/validation.py

```diff
@@ -115,7 +115,13 @@ def disallow_extra_fields(self, processed_data, original_data):
     RequestSchema = Schema
 
 
+# Define a constructor to please some IDE type checking
 class ResponseSchema(ActuallyRequireOnDumpMixin, Schema):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+
+class StrictSchema(RequestSchema, ResponseSchema):
```

Comment:
- Better to inherit the mixin classes directly
- I wonder if there’s a better name than StrictSchema for this, since strict is overloaded with other meaning in Marshmallow v2 (complicated by how Rebar currently already always does a strict load with unstrict Marshmallow v2 schemas, i.e. raising rather than returning a value with errors in it)